frontend/src/pages/LearningCourse/FormDialog/FormDialog.tsx

```diff
@@ -43,35 +39,27 @@ const FormDialog: React.FC<IFormDialog> = ({ dialogOpen, handleDialogClose }) =>
 
   const handleStartTest = () => {
     startTestMutate(params.courseId);
+    navigateTo(`${PATHS.learnCourse}/${params.courseId}/test`);
   };
 
   return (
```

Comment:
In case it's a component, remove please all the logic(functions) to the container please

Reply:
done